_posts/2021-07-01-pulpcore-3.14-release-announcement.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Pulp 3.14 is now Generally Available
+author: Melanie Corr
+tags:
+  - release
+---
+
+# 3.14 - headline features
+
+We are happy to announce the release of Pulpcore 3.14!
+
+This release contains a number of bug fixes and enhancements. In this blog, we'll take a look at
+the headline features.
+
+A gentle reminder that the [Community Survey](https://forms.gle/KJDoSdqWf5ajut5f6) is open,
+and we would love to hear from you.
+
+## Pulp has a new queueless tasking system! [#8501](https://pulp.plan.io/issues/8501)  
+
+The largest and most significant change introduced with this release is the new tasking system.
+ With this change, Pulp no longer needs a resource manager.
+
+If you'd like an in-depth look at the motivations, benefits, and details of the new tasking system,
+read [Matthias Dellweg's blog post](https://pulpproject.org/2021/06/21/a-new-tale/) and also watch
+ a YouTube livestreamed interview with Matthias conducted by Pulp's Brian Bouterse:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YWKw4RYluPM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Note that for the moment, this feature is in tech-preview.
+Please [let us know what you think](https://pulpproject.org/help/#github-discussions).
+
+### Sticking with the old tasking system?
+If you intend to keep using the old tasking system, you should configure the `USE_NEW_WORKER_TYPE`
+setting to `false` before upgrading. For more information about how to enable or disable the new
+tasking system, see the [documentation](https://docs.pulpproject.org/pulpcore/configuration/settings.html#use-new-worker-type).
+
+
+## New Orphan Cleanup Endpoint [#8658](https://pulp.plan.io/issues/8658)
+
+This release introduces the `/pulp/api/v3/orphans/cleanup/` endpoint.
+When you call this endpoint with `POST` and no parameters, it is the equivalent of calling.
+`DELETE /pulp/api/v3/orphans/`. Note that as part of this release,
+`DELETE /pulp/api/v3/orphans/` has been deprecated.
+
+You can specify a list of `content_hrefs` to delete using the optional parameter `content_hrefs`
+with list of content hrefs. In this case, only the specified content units will deleted by the
+orphan cleanup.
```

Comment:
```suggestion
with list of content hrefs. In this case, only the specified content units will be deleted by the
orphan cleanup.
```
Maybe even better: "...will be considered for deletion"

Reply:
My fire alarm went off (falsely) at 2.30am last night. I read through this about fifty times but still am making stupid mistakes; I apologise that you have to fix these things for me.